tests/backends/stream/test_ws.py

```diff
@@ -17,10 +16,6 @@ def test_backends_stream_ws_stream_instantiation(ws):
 
     assert WSStream.name == "ws"
 
-    error = "missing 1 required positional argument: 'uri'"
-    with pytest.raises(TypeError, match=error):
-        WSStream()  # pylint: disable=no-value-for-parameter
```

Comment:
Why removing this test?

Reply:
Good catch) Thanks) The WSStream constructor signature was updated with a default value. Updated this test)